src/app/modules/shoppingCart/services/shopping-cart.service.ts

```diff
@@ -34,4 +34,24 @@ export class ShoppingCartService {
         }
       });
   }
+
+  deleteFromCart(id: string) {
+    let cartId = localStorage.getItem('cartId');
+    return this.db.object(`/shoppingCart/${cartId}/items/${id}`).remove();
+  }
+
+  getListItemsShoppingCart() {
+    let cartId = localStorage.getItem('cartId');
+    return this.db
+      .list(`/shoppingCart/${cartId}/items/`)
+      .snapshotChanges()
+      .pipe(
+        map(courses =>
+          courses.map(c => ({
```

Comment:
spell out course. avoid using abbreviations because they generally confuse devs who are reading the code. The extra characters are worth the clarity.